lambdaclass/curse_of_myrra
# Character info screen

Closes #1010 

## Motivation

We needed to implement the character info screen where you can see all characters information and select it fo the future game flow.

## Summary of changes
I branched from `gh-926-character-list`
- Implement a new Scene, CharacterInfo
- Scripts:
	- Created `CharacterInfoManager`
	- Changed `MainScreenModelManager` to `UIModelManager`
	- Changed `SkillDescription`
	- Added the fields to `ComCharacter` and `SkillInfo`
	- Created `CharacterListItem.cs`
- Changed `CharactersListCameraUI` title

## How has this been tested?

1. Start the game in te MainScreen scene.
2. Click in characters
3. Select any character, you should see the character info screen of the selected character
4. Click the abilities and see if their change
5. Click the left and right arrows and see the characters change
6. Use the back button and repetear the process

## Test additions / changes

List tests added/updated.

## Checklist


![Captura de pantalla 2023-11-01 a la(s) 16 47 15](https://github.com/lambdaclass/curse_of_myrra/assets/62818001/5f3b3edc-b979-4d8c-ab09-0f64fb69ba32)
